### ISSUE TITLE
Update modal styling

### ISSUE THREAD

**Issue by bwmarkle** [W3EDGE/w3-total-cache#206]
To something like this:
![image](https://user-images.githubusercontent.com/5925563/84931140-60973680-b0a0-11ea-9be7-c59b1e35ba4d.png)

**Comment by bwmarkle** [comment]
@maxicus 

Are these `#w3tc-upgrade ul` styles old, can we get rid of them? Example:
```css
.w3tc-overlay #w3tc-upgrade ul {
    width: 480px;
    float: right;
    margin-right: 20px;
}
.w3tc-overlay #w3tc-upgrade li {
    background: url(../img/overlay/list-check.png) no-repeat 0px 3px;
    color: #444;
    list-style-type: none;
    padding: 0 0 1px 20px;
}
```

---

Besides the upgrade modal, are there any other modals in the plugin using `w3tc-overlay`?

**Comment by maxicus** [comment]
@bwmarkle .w3tc-overlay is used by all overlays, 
.w3tc-overlay #w3tc-upgrade only by this upgrade overlay - so if this overlay looks ok after the change - it's a safe change =)

**Comment by bwmarkle** [comment]
Closing. Merged into 0.14.3-branch in this PR https://github.com/W3EDGE/w3-total-cache/pull/211
